cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -467,6 +473,17 @@ pub mod test_helpers {
         let file = dir.join(file_name);
         fs::read_to_string(file).unwrap()
     }
+
+    /// Assert that a build succeeds given the path to the environment
+    pub fn assert_manifest_build_succeeds(path: impl AsRef<Path>, name: &str) {
+        let path = path.as_ref();
+        let (flox, tmpdir) = flox_instance();
+        copy_dir_recursive(path, &tmpdir, true).unwrap();
+        let ConcreteEnvironment::Path(mut env) = open_path(&flox, &tmpdir).unwrap() else {
+            panic!("expected path environment")
+        };
+        assert_build_status(&flox, &mut env, name, None, true);
+    }
 }
```

Comment:
suggestion: I'd prefer consistency with `assert_build_status` and take `&flox` from the test impl, rather than producing it here.

Reply:
Fixed